Explorer/Assets/DCL/ExplorePanel/ExplorePanelController.cs

```diff
@@ -1,47 +1,68 @@
+using Arch.Core;
 using Cysharp.Threading.Tasks;
 using DCL.Backpack;
+using DCL.CharacterMotion.Components;
 using DCL.Navmap;
 using DCL.Settings;
 using DCL.UI;
 using MVC;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading;
+using UnityEngine;
+using UnityEngine.InputSystem;
 using Utility;
 
 namespace DCL.ExplorePanel
 {
     public class ExplorePanelController : ControllerBase<ExplorePanelView, ExplorePanelParameter>
     {
+        // Is there another way?
+        private static ExploreSections lastShownSection;
+
+        public static ExploreSections GetLastShownSection() =>

```

Comment:
You use it from the same plugin you instantiate the controller itself, simple save it in a variable in the plugin and you will be able to get rid of `static`